pipeline/train/spm-vocab.sh

```diff
@@ -70,6 +80,17 @@ zstdmt -dc "${merged_corpus_trg}" >"${vocab_dir}/data.trg.txt"
 # byte_fallback - decomposes unknown pieces into UTF-8 bytes
 # user_defined_symbols - placeholders
 # character_coverage - CJK is recommended to have 0.9995, vocab languages probably you want 1
+if [[ ${SRC} == 'zh' ||
+      ${TRG} == 'zh' ||
+      ${SRC} == 'ja' ||
+      ${TRG} == 'ja' ||
+      ${SRC} == 'ko' ||
+      ${TRG} == 'ko' ]]
+then
+  character_coverage=0.9995
+else
+  character_coverage=1.0
+fi
```

Comment:
As I said in #745, I think default coverage would be enough for all languages if byte fallckback is enabled.

Reply:
ok, replaced with sp default (removed the argument)